lighthouse-core/fraggle-rock/gather/navigation-runner.js

```diff
@@ -85,7 +85,7 @@ async function _afterTimespanPhase(navigationContext, artifacts) {
     if (!gatherer.meta.supportedModes.includes('timespan')) continue;
 
     const artifactPromise = (artifacts[artifactDefn.id] || Promise.resolve()).then(() =>
-      gatherer.afterTimespan({driver: navigationContext.driver, gatherMode: 'navigation'})
+      gatherer.afterTimespan({driver: navigationContext.driver, gatherMode: 'navigation', dependencies: /** @type {LH.Artifacts} */ (artifacts)})
```

Comment:
IRL, this will be narrowed down to the set that is requested (and has been validated) by the config in `gatherer.meta`